tests/target/issue-2672.rs

```diff
@@ -0,0 +1,21 @@
+fn main() {
+    fn test(
+        averylongname_asdfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffdsffffffffffffffffffffffffffffffffffffffffffffffff: i32,
+    ) {
+    }
+    fn test(
+        shortname: i32,
+        averylongname_asdfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffdsffffffffffffffffffffffffffffffffffffffffffffffff: i32,
+    ) {
+    }
+
+    let averylongname_asdfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff =
+        0;
+}
+
+mod asdf {
+    fn test(
+        averylongname_asdfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffdsffffffffffffffffffffffffffffffffffffffffffffffff: i32,
+    ) {
+    }
+}
```

Comment:
Could we include the other snippets posted in https://github.com/rust-lang/rustfmt/issues/2672#issuecomment-416498411 on that thread?